provisioning/cmd/provisioning-service/main.go

```diff
@@ -73,8 +74,19 @@ func main() {
 	provisioner := provisioner.New(vtsClient)
 
 	log.Infow("initializing provisioning API service", "address", cfg.ListenAddr)
+	authorizer, err := auth.NewAuthorizer(subs["auth"], log.Named("auth"))
```

Comment:
Presumably if the Line 41 `auth` subs is missing the NewAuthroizer will default to `passthrough` ?

Reply:
Yup: https://github.com/veraison/services/blob/setrofim-auth/auth/authorizer.go#L21